_data/en/settings.yml

```diff
@@ -2,8 +2,8 @@ accessible_labels:
   login_link_logo: Login.gov Homepage
   partners_link_logo: Login.gov Partners Homepage
   footer_navigation: Footer navigation
-  primary_navigation: Primary navigation
-  secondary_navigation: Secondary navigation
+  primary_navigation: Primary links navigation
```

Comment:
As I understand it, the problem was that the label including "navigation" will cause the screen reader to read as "primary navigation navigation", repeating "navigation". This will still be a problem after these changes, it'll just be "primary links navigation navigation". 

I think we wanted to just have "Primary links" ?

Reply:
got ya. I misunderstood this https://cm-jira.usa.gov/browse/LG-9626 where it said it needed to be "primary links" or secondary links

Reply:
Im curious will it translate this properly "navigation" if we remove navigation"

Reply:
The second "navigation" comes from the screen reader, so that would depend how the screen reader announces it, which may be based on the user's system settings. I would expect if someone's browsing the French version of the site and has their system configured in French, it would read it all in French, yes.

Reply:
awesome made the change so can we review this when u get a chacne.